selector/random/random.go

```diff
@@ -0,0 +1,10 @@
+package random
+
+import (
+	"github.com/micro/go-micro/v2/selector"
+)
+
+// NewSelector returns a random selector
+func NewSelector(opts ...selector.Option) selector.Selector {
+	return selector.DefaultSelector
```

Comment:
is this not a bit fragile? assumes `selector.DefaultSelector` is a `random` ?

Reply:
Unfortunately this is a standard practice in go-micro because of the DefaultX pattern. Each package has a default which has to be implemented at a package level to prevent a circular dependancy. It's something which needs to be amended globally, there is a PR @vtolstov opened which was working towards this, but it's not yet been merged.